### PR TITLE
[SELC-5608] FEAT: Added new institutionType GPU and attached radio button

### PR DESCRIPTION
**Pull request by GabrieleMaiocchiFilippo**
#### List of Changes

[SELC-5608] FEAT: Added new institutionType GPU and attached radio button

#### Motivation and Context

Added new InstitutionType GPU and attacched radioButton to open a new onboarding flow

#### How Has This Been Tested?

- Tested that the new institutionType and the new radioButton is present in the institution type choice for the correct products
- Added junit test for the new institutionType for the correct products

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

[SELC-5608]: https://pagopa.atlassian.net/browse/SELC-5608?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/__tests__/StepInstitutionType.test.tsx [hunk 1]
```diff
@@ -30,13 +30,13 @@ test('Test: The correct institution types with the expected descriptions, can be
 
       switch (p.id) {
         case 'prod-pagopa':
-          expect(institutionTypeValues).toStrictEqual(['PA', 'GSP', 'PT', 'PSP', 'PRV']);
+          expect(institutionTypeValues).toStrictEqual(['PA', 'GSP', 'PT', 'PSP', 'PRV', 'GPS',]);
```
Comment (andrea-putzu) [inline src/components/__tests__/StepInstitutionType.test.tsx:33]: is it a typo?

File: src/utils/constants.ts [hunk 2]
```diff
@@ -244,6 +245,7 @@ export const institutionType4Product = (productId: string | undefined) => {
         (it) =>
           it.labelKey === 'pa' ||
           it.labelKey === 'gsp' ||
+          it.labelKey === 'gps' ||
```
Comment (andrea-putzu) [inline src/utils/constants.ts:248]: is it a typo?

File: src/locale/it.ts [hunk 3]
```diff
@@ -573,6 +573,10 @@ export default {
         title: 'Gestore di servizi pubblici',
         description: 'art. 2, comma 2, lettera B del CAD',
       },
+      gps: {
```
Comment (andrea-putzu) [inline src/locale/it.ts:576]: is it a typo?

File: src/components/__tests__/StepInstitutionType.test.tsx [hunk 4]
```diff
@@ -30,13 +30,13 @@ test('Test: The correct institution types with the expected descriptions, can be
...
           break;
         case 'prod-io':
-          expect(institutionTypeValues).toStrictEqual(['PA', 'GSP', 'PT']);
+          expect(institutionTypeValues).toStrictEqual(['PA', 'GSP', 'PT', 'GPS']);
           break;
         case 'prod-io-sign':
-          expect(institutionTypeValues).toStrictEqual(['PA', 'GSP']);
+          expect(institutionTypeValues).toStrictEqual(['PA', 'GSP', 'GPS',]);
```
Comment (andrea-putzu) [inline src/components/__tests__/StepInstitutionType.test.tsx:39]: is it a typo?

File: src/components/__tests__/StepInstitutionType.test.tsx [hunk 5]
```diff
@@ -30,13 +30,13 @@ test('Test: The correct institution types with the expected descriptions, can be
 
       switch (p.id) {
         case 'prod-pagopa':
-          expect(institutionTypeValues).toStrictEqual(['PA', 'GSP', 'PT', 'PSP', 'PRV']);
+          expect(institutionTypeValues).toStrictEqual(['PA', 'GSP', 'PT', 'PSP', 'PRV', 'GPS',]);
           break;
         case 'prod-io':
-          expect(institutionTypeValues).toStrictEqual(['PA', 'GSP', 'PT']);
+          expect(institutionTypeValues).toStrictEqual(['PA', 'GSP', 'PT', 'GPS']);
```
Comment (andrea-putzu) [inline src/components/__tests__/StepInstitutionType.test.tsx:36]: is it a typo?

File: src/utils/constants.ts [hunk 6]
```diff
@@ -252,12 +254,13 @@ export const institutionType4Product = (productId: string | undefined) => {
...
           (ENV.ENV !== 'PROD' && it.labelKey === 'psp') ||
           (ENV.PT.SHOW_PT ? it.labelKey === 'pt' : '') ||
           (ENV.PURE_PRV.SHOW ? it.labelKey === 'oth' : '')
       );
     case 'prod-io-sign':
-      return institutionTypes.filter((it) => it.labelKey === 'pa' || it.labelKey === 'gsp');
+      return institutionTypes.filter((it) => it.labelKey === 'pa' || it.labelKey === 'gsp' || it.labelKey === 'gps');
```
Comment (andrea-putzu) [inline src/utils/constants.ts:263]: is it a typo?

File: types.ts [hunk 7]
```diff
@@ -247,7 +247,7 @@ export type InstitutionOnboardingInfoResource = {
   institution: InstitutionData;
 };
 
-export type InstitutionType = 'PA' | 'GSP' | 'SCP' | 'PT' | 'PSP' | 'SA' | 'AS' | 'PRV';
+export type InstitutionType = 'PA' | 'GSP' | 'SCP' | 'PT' | 'PSP' | 'SA' | 'AS' | 'PRV' | 'GPS';
```
Comment (andrea-putzu) [inline types.ts:250]: is it a typo?

File: src/utils/constants.ts [hunk 8]
```diff
@@ -275,6 +278,8 @@ export const description4InstitutionType = (institutionType: { labelKey: string;
       return 'stepInstitutionType.institutionTypes.gsp.description';
     case 'SCP':
       return 'stepInstitutionType.institutionTypes.scp.description';
+    case 'GPS':
+      return 'stepInstitutionType.institutionTypes.gps.description';
```
Comment (andrea-putzu) [inline src/utils/constants.ts:282]: is it a typo?

File: src/components/__tests__/StepInstitutionType.test.tsx (hunk 1 above)
Comment (GabrieleMaiocchiFilippo) [reply src/components/__tests__/StepInstitutionType.test.tsx:33]: yes i meant GPU

File: src/locale/it.ts (hunk 3 above)
Comment (GabrieleMaiocchiFilippo) [reply src/locale/it.ts:576]: yes i meant gpu

File: src/components/__tests__/StepInstitutionType.test.tsx (hunk 4 above)
Comment (GabrieleMaiocchiFilippo) [reply src/components/__tests__/StepInstitutionType.test.tsx:39]: yes i meant GPU

File: src/components/__tests__/StepInstitutionType.test.tsx (hunk 5 above)
Comment (GabrieleMaiocchiFilippo) [reply src/components/__tests__/StepInstitutionType.test.tsx:36]: yes i meant GPU

File: src/utils/constants.ts (hunk 6 above)
Comment (GabrieleMaiocchiFilippo) [reply src/utils/constants.ts:263]: yes i meant gpu

File: types.ts (hunk 7 above)
Comment (GabrieleMaiocchiFilippo) [reply types.ts:250]: yes i meant GPU

File: src/utils/constants.ts (hunk 8 above)
Comment (GabrieleMaiocchiFilippo) [reply src/utils/constants.ts]: yes i meant GPU

**Review by andrea-putzu — APPROVED** [review]

**Pull request merged**